### DISCUSSION TITLE
[draft-v1] Figure 2: Number of committers

### DISCUSSION THREAD

**Discussion by pllim** [astropy/astropy-v5.0-paper#93] [Figure comments] [answered]
Probably no way we can get the number of users for each era, but if we had the numbers, I suspect the ratio of user/committer has exponential growth; i.e., smaller number of people supporting a much larger user base over time. 💭

**Comment by adrn** [comment, marked as answer]
That would be interesting! I think it could be possible to get pypi download stats back to 2017: https://packaging.python.org/en/latest/guides/analyzing-pypi-package-downloads/
but I don't have time to incorporate this into this draft of the paper. I opened an issue [here](https://github.com/adrn/astropy-sustainability/issues/8) to remember this point (I still am slowly putting together a sustainability dashboard for astropy that gets updated automatically and regularly, so that would be a natural place to put it). Does that sound ok?
